### PR TITLE
Weizhuoz/auto guilty commit

### PR DESCRIPTION
**Pull request by WeizhuoZhang-intel**
- [x] Generate model list for guilty commit search job
- [x] Create a groovy for Auto guilty commit search

**Step 1**: `report.py` will generate 2 files which are `guilty_commit_search_model_list.csv` and `guilty_commit_search_model_list.json`. Those files contain the following infomation

- **suite**: torchbench, huggingface, timm_models
- **model** name
- **scenario**: performance or accuracy
- **thread mode**: multiple or single
- **test kind**: crash, drop, fixed or improve
- **precision**

**Step 2**: [inductor_aws_guilty_commit_job_trigger](https://inteltf-jenk.sh.intel.com/view/PT_Inductor/job/inductor_aws_guilty_commit_job_trigger/) will use `guilty_commit_search_model_list.json` to launch auto guilty commit jobs parralelly

### CODE REVIEW

**Comment by WeizhuoZhang-intel** [comment]
Hi @chuanqi129 ,
The version collection steps has been moved in Dockfile build steps.


And the version file has been stored in csv format seperated by `,` as `version.csv`
Title is `name,branch,commit`
image

File: groovy/inductor_guilty_commit_job_trigger.groovy [hunk 1]
```diff
@@ -0,0 +1,36 @@
+node("weizhuoz-mlt-ace"){
```
Comment (chuanqi129) [inline groovy/inductor_guilty_commit_job_trigger.groovy:1]: parameterize the node

File: groovy/inductor_guilty_commit_job_trigger.groovy [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        def job_list = [:]
+        for (i = 0; i < size; i += 1) {
+            def elem = lines[i]
+            if (test_kinds.contains(elem['kind'])) {
+                def instance_name = 'icx-guilty-search'
+                if (elem['precision'] == "amp") {
+                    instance_name = 'spr-guilty-search'
+                }
+                job_list["Job_${i}"] = {
+                    guilty_commit_search_job = build job: 'inductor_aws_guilty_commit_search_zwz', propagate: false, parameters: [
```
Comment (chuanqi129) [inline groovy/inductor_guilty_commit_job_trigger.groovy:16]: change to the norm one

File: groovy/inductor_guilty_commit_job_trigger.groovy [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+                        [$class: 'StringParameterValue', name: 'model', value: elem['name']],
+                        [$class: 'StringParameterValue', name: 'scenario', value: elem['scenario']],
+                        [$class: 'StringParameterValue', name: 'kind', value: elem['kind']],
+                        [$class: 'StringParameterValue', name: 'TORCH_START_COMMIT', value: TORCH_START_COMMIT],
+                        [$class: 'StringParameterValue', name: 'TORCH_END_COMMIT', value: TORCH_END_COMMIT],
+                    ]
+                }
+            }
+        }
+        parallel job_list
```
Comment (chuanqi129) [inline groovy/inductor_guilty_commit_job_trigger.groovy:34]: Will this launch job collect all sub job results and summary those search results by guilty commit?

Comment (WeizhuoZhang-intel) [reply groovy/inductor_guilty_commit_job_trigger.groovy:34]: Not yet, will do this later.

**Pull request closed without merging**